aospy_synthetic/test/heirarchical_test_objs.py

```diff
@@ -0,0 +1,33 @@
+class TestProj(object):
```

Comment:
This needs a module-level docstring, even just a sentence or two, because its not self-evident from the code what this is for.

(Strictly speaking, all modules should have a docstring... :))